applications/plugins/SofaPython/Binding_SofaModule.cpp

```diff
@@ -611,4 +851,14 @@ SP_MODULE_METHOD(Sofa,loadPythonSceneWithArguments)
 SP_MODULE_METHOD(Sofa,loadPlugin)
 SP_MODULE_METHOD(Sofa,path)
 SP_MODULE_METHOD(Sofa,getAvailableComponents)
+SP_MODULE_METHOD_DOC(Sofa, timerClear, "Method : Sofa_clear \nDesc   : Wrapper for python usage. Clear the timer. \nParam  : PyObject*, self - Object of the python script \nReturn : return NULL")
```

Comment:
The docs string here are using *pointer and NULL which are c++ terminology which have no meaning in the python context.